vertica/datadog_checks/vertica/queries.py

```diff
@@ -5,28 +5,53 @@
 # Using System tables to monitor Vertica:
 # https://www.vertica.com/docs/11.1.x/HTML/Content/Authoring/AdministratorsGuide/Monitoring/Vertica/UsingSystemTables.htm
 
-from collections import OrderedDict
 from functools import partial
 
-
-def get_queries(major_version, metric_groups):
-    """Get query dicts for the given metric_groups for use with QueryManager."""
-    queries = []
-
-    metric_groups = set(metric_groups).intersection(METRIC_GROUPS)
-
-    for metric_group in metric_groups:
-        queries.extend(METRIC_GROUPS[metric_group](major_version))
-
-    return queries
-
-
-def build_licenses_queries(version=None):
-    """
-    https://www.vertica.com/docs/11.1.x/HTML/Content/Authoring/SQLReferenceManual/SystemTables/CATALOG/LICENSES.htm
-    """
-    name = 'licenses'
-    query = """
+# If you create a new group, please add it to `METRIC_GROUPS` below and
+# the config file (under `metric_groups`). Also add a `build_{group_name}_queries`
+# method to the QueryBuilder that returns the associated list of queries.
```

Comment:
We might be able to add a test to ensure that's done. What do you think?

Reply:
I believe that tests already do this, since the default is to take all groups in `METRIC_GROUPS`

https://github.com/DataDog/integrations-core/blob/13937d185b1beeecd624102bfe2c3de09246e88b/vertica/datadog_checks/vertica/vertica.py#L139-L140

And `test_check` tests the default. Thus, if any group name didn't have a corresponding `build_` method, the test would complain about the missing method.

Is that what you meant or is there something else I'm missing?

Reply:
Oh ok you're right! Thanks :)